website/docs/cloud-docs/integrations/run-tasks/index.mdx

```diff
@@ -17,6 +17,10 @@ To build a custom integration, you must have a server capable of receiving reque
 
 This feature relies heavily on the proper parsing of [plan JSON output](/internals/json-format). When sending this output to an external system, be certain that system can properly interpret the information provided.
 
+## Available Run Tasks
+
+-> **Note:** You can view the most up-to-date list of run tasks in the [Terraform Registry](https://registry.terraform.io/browse/run-tasks).
```

Comment:
```suggestion
You can view the most up-to-date list of run tasks in the [Terraform Registry](https://registry.terraform.io/browse/run-tasks).
```

Since we're removing the list of partners, this part shouldn't be just a note anymore.